### PR TITLE
OWASP MASVS v2 compliance integration

### PR DESCRIPTION
**Pull request by future-pirate-king**

### CODE REVIEW

File: app/components/edit-analysis-details/index.hbs [hunk 1]
```diff
@@ -309,6 +309,28 @@
           </PowerSelectMultiple>
         </div>
       </div>
+      <div class='columns'>
+        <div class='column is-one-third'>
+          <h5 class='bold-font'>
+            OWASP MASVS (v2) Requirements
```
Comment (Yibaebi) [inline app/components/edit-analysis-details/index.hbs:315]: Could this be translated?

File: app/components/edit-analysis-details/index.hbs [hunk 2]
```diff
@@ -309,6 +309,28 @@
...
+          </h5>
+        </div>
+      </div>
+      <div class='columns border-bottom-container'>
+        <div class='column margin-bottom20'>
+          <PowerSelectMultiple
+            @placeholder='Select OWASP MASVS (v2) Requirements'
```
Comment (Yibaebi) [inline app/components/edit-analysis-details/index.hbs:322]: Could this be translated?

File: app/components/edit-analysis-details/index.hbs [hunk 3]
```diff
@@ -309,6 +309,28 @@
           </PowerSelectMultiple>
         </div>
       </div>
+      <div class='columns'>
+        <div class='column is-one-third'>
+          <h5 class='bold-font'>
```
Comment (Yibaebi) [inline app/components/edit-analysis-details/index.hbs:314]: Also, can we use `AkTypography` here?

File: app/components/edit-analysis-details/index.hbs (hunk 1 above)
Comment (future-pirate-king) [reply app/components/edit-analysis-details/index.hbs:315]: resolved internally

File: app/components/edit-analysis-details/index.hbs (hunk 2 above)
Comment (future-pirate-king) [reply app/components/edit-analysis-details/index.hbs:322]: resolved internally

File: app/components/edit-analysis-details/index.hbs (hunk 3 above)
Comment (future-pirate-king) [reply app/components/edit-analysis-details/index.hbs:314]: resolved internally

**Pull request merged**